test/common.js

```diff
@@ -402,8 +402,10 @@ function runCallChecks(exitCode) {
 }
 
 
-exports.mustCall = function(fn, expected) {
-  if (typeof expected !== 'number') expected = 1;
+exports.mustCall = function(fn, expected = 1) {
```

Comment:
It would really be nice to have the `common.mustCall` change in a separate PR (including the fixes to tests that don't pass a number). Then when it lands you can rebase this one.